test/testsuite/lantest.c

```diff
@@ -150,6 +151,26 @@ static const char *test_names[NUMTESTS] = {
     "Cache validation efficiency (metadata changes) [30,000 AFP ops]"  /* TEST_CACHE_VALIDATION */
 };
 
+/* Error type descriptions for clean_exit() */
+typedef struct {
+    uint8_t exit_code;
+    const char *category;
+    const char *description;
+} ErrorDescription;
+
+static const ErrorDescription error_descriptions[] = {
+    {0, "Normal Exit", "Clean shutdown"},
+    {2, "Memory Allocation", "Buffer/cache allocation failures"},
+    {3, "File/Directory Ops", "Create/delete/access failures"},
+    {4, "Fork Operations", "Fork open/close failures"},
+    {5, "Network/Protocol", "AFP protocol/read/write errors"},
+    {6, "Thread Operations", "pthread_create failures"},
+    {7, "Validation/Test", "Test validation errors"},
+    {8, "Configuration", "Invalid parameters/out of range"},
+    {9, "Volume Operations", "Volume open/access failures"},
```

Comment:
How about you define macro constants for the error codes, f.e.:

```
#define AFPTEST_NORMAL_EXIT 0
...
#define AFPTEST_CONFIGURATION_ERROR 8
```

...then you don't have to lean on code comments to explain the magic numbers:

```
        clean_exit(AFPTEST_CONFIGURATION_ERROR);
```

What do you think?

Reply:
Done